staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/webhook/admission.go

```diff
@@ -40,10 +41,11 @@ import (
 	genericadmissioninit "k8s.io/apiserver/pkg/admission/initializer"
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
-	admissioninit "k8s.io/kubernetes/pkg/kubeapiserver/admission"
+)
 
-	// install the clientgo admission API for use with api registry
-	_ "k8s.io/kubernetes/pkg/apis/admission/install"
+const (
+	// Name of admission plug-in
+	PluginName = "GenericAdmissionWebhook"
```

Comment:
nit: just WebHook?

Reply:
ok, that name is not new. So we have to stick. Please change the name in `Register` below to use the constant.